Monika After Story/game/zz_backgrounds.rpy

```diff
@@ -0,0 +1,491 @@
+#Here's where we store our background data
+default persistent._mas_background_MBGdata = {}
+
+#START: Class definition
+init -10 python:
+    class MASBackground(object):
+        """
+        Background class to get display props for bgs
+
+        PROPERTIES:
+            background_id - the id which defines this bg
+            prompt - button label for the bg
+            img_map - Dict mapping all images for the bgs, keys are precip types (See MASWeather)
```

Comment:
```suggestion
            image_map - Dict mapping all images for the bgs, keys are precip types (See MASWeather)
```
minor thing, but since the property is `image_map`, keeping class documentation consistent with actual names makes searching for stuff easier.